parley/src/resolve/mod.rs

```diff
@@ -152,7 +152,7 @@ impl ResolveContext {
             }
             StyleProperty::StrikethroughSize(value) => StrikethroughSize(value.map(|x| x * scale)),
             StyleProperty::StrikethroughBrush(value) => StrikethroughBrush(value.clone()),
-            StyleProperty::LineHeight(value) => LineHeight(*value),
+            StyleProperty::LineHeight(value) => LineHeight(value.scale(scale)),
```

Comment:
This is correct and is in the `resolve_property` method.

However, the next method, `resolve_entire_style_set`, needs the following edit:

```diff
- line_height: raw_style.line_height,
+ line_height: raw_style.line_height.scale(scale),
```